src/tiler/package.json

```diff
@@ -6,8 +6,12 @@
   "dependencies": {
     "windshaft": "0.36.0"
   },
+  "devDependencies": {
+    "supervisor": "^0.6.0"
```

Comment:
According to [this project's `README`](https://github.com/isaacs/node-supervisor/blob/master/README.md), it is no longer being maintained. Do we have any better alternatives?